addon/components/nrg-datetime/calendar/component.js

```diff
@@ -178,6 +179,7 @@ export default Component.extend(EKMixin, EKFirstResponderOnFocusMixin, {
           date,
           disabled,
           selected,
+          context: (isDifferentMonth && {unitType: 'month', number: monthIndex}) || '',
```

Comment:
What if you just pass `month` instead of context?

```js
const month = calendar.month();
...
{
  ...
  month,
  ...
}
```

Instead of making the changes in `_manipulateDate`, you can do something like this in the `clickCell` function.
```js
this._manipulateDate('set', cell.date, 'date');
this._manipulateDate('set', cell.month, 'month');
```